lib/govuk_publishing_components/presenters/intervention_helper.rb

```diff
@@ -16,10 +17,10 @@ def accessible_text
       end
 
       def dismiss_link
-        if @query_string.present?
-          "#{@query_string}&hide-intervention=true"
-        else
+        if !@query_string.present?
           "?hide-intervention=true"
+        elsif !@query_string.include?("hide-intervention=true")
+          "#{@query_string}&hide-intervention=true"
```

Comment:
I think this change can be removed completely. It's doing the same as the original if/else, except it's doing a negative check.

i.e. it's the same as leaving it as:

```ruby
if @query_string.present?
   "#{@query_string}&hide-intervention=true"
else
   "?hide-intervention=true"
end
```

Reply:
🤦